tests/unit/samplegen/test_samplegen.py

```diff
@@ -0,0 +1,352 @@
+# Copyright (C) 2019  Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from collections import namedtuple
+
+import unittest
+import gapic.samplegen.samplegen as samplegen
+import gapic.samplegen.utils as utils
+
+
+class TestValidateResponse(unittest.TestCase):
```

Comment:
The rest of the library is using pytest; can you follow suit?

Basically these become their own functions:

```python
def test_define():
    define = {'define': 'squid=$resp'}
    assert samplegen.validate_response([define])
```

Reply:
Will do.

Reply:
Woah, pytest is so...straightforward and...pleasant.